src/components/Login.js

```diff
@@ -120,8 +119,8 @@ class Login extends Component {
                                 </div>
                               </div>
                             </div>
-                            <div className="footer text-center">
-                              <button onClick={this.submitForm} className="btn btn-info btn-round btn-wd btn-lg">Let's go</button>
+                            <div className="footer text-center" style={{display:'flex', alignItems: 'center', justifyContent: 'center'}}>
+                              {this.props.auth.isFetching ? <ReactLoading type={"spin"} height={20} width={20} color="#0000ff"/> : <button onClick={this.submitForm} className="btn btn-info btn-round btn-wd btn-lg">Let's go</button>}
```

Comment:
I prefer this syntax:

```
{
  this.props.auth.isFetching 
  ? <ReactLoading type={"spin"} height={20} width={20} color="#0000ff"/> 
  : <button onClick={this.submitForm} className="btn btn-info btn-round btn-wd btn-lg">
      Let's go
    </button>
}
```

because it's much more legible.

Reply:
nice. made the change.